studio/components/to-be-cleaned/SqlEditor/TabSqlQuery.js

```diff
@@ -289,7 +289,7 @@ const UtilityTabResults = observer(() => {
     return (
       <div className="bg-table-header-light dark:bg-table-header-dark">
         <p className="m-0 border-0 px-6 py-4 text-sm text-scale-1000">
-          Click <code>RUN</code> to execute your query.
+          Click <code>RUN</code> or hit <code>cmd/ctrl + return</code> to execute your query.
```

Comment:
nit: can we programtically show cmd or ctrl depending on the OS? also use unicode icon for "return"

Reply:
Nice!  Good idea. Added.